### PR TITLE
[Adjustment][Maxell] Homepage, Link, and Config Refactor

### PR DESCRIPTION
**Pull request by maxellmilay**
Changes:
 - added external links and internal routes to buttons
 - stored links in enums
 - added new UI in home page social media
 - changed color config
 - made section mobile responsive

New Social Media Section UI:
image

Mobile Version:
image
image

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -11,18 +14,44 @@ const HomePage = () => {
...
+            </a>
+          </div>
+          <div className="flex flex-col items-center justify-center font-bold bg-mingor-gray rounded-[30px] h-1/2 px-10">
+            <p className="text-mingor-blue-2 text-6xl mb-10">
+              Send us a message!
+            </p>
+            <a
+              href={externalLinks.PALANCA}
+              target="_blank"
+              className="text-white bg-mingor-blue-2 px-10 py-2 text-2xl w-fit rounded-[50px]"
```
Comment (ezerssss) [inline src/app/page.tsx]: dili need ug

```
rel=noreferrer?
```

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -9,15 +10,19 @@ const Footer = () => {
       </p>
       <p className="text-xs mb-3">Gayang Mingor © 2023</p>
       <div className="flex md:text-3xl">
-        <button className="mr-5 md:mr-10">
+        <a href={externalLinks.FB} target="_blank" className="mr-5 md:mr-10">
```
Comment (ezerssss) [inline src/components/Footer.tsx]: ari pod? noreferer

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -9,15 +10,19 @@ const Footer = () => {
...
           <FaFacebook />
-        </button>
-        <button className="mr-5 md:mr-10">
+        </a>
+        <a
+          href={externalLinks.TWITTER}
+          target="_blank"
+          className="mr-5 md:mr-10"
+        >
```
Comment (ezerssss) [inline src/components/Footer.tsx]: here

File: src/components/Footer.tsx [hunk 4]
```diff
@@ -9,15 +10,19 @@ const Footer = () => {
...
           <FaTwitter />
-        </button>
-        <button>
+        </a>
+        <a href={externalLinks.INSTAGRAM} target="_blank">
```
Comment (ezerssss) [inline src/components/Footer.tsx]: here

File: tailwind.config.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ module.exports = {
       colors: {
         "mingor-yellow": "#E9C717",
         "mingor-blue": "#14194A",
-        "mingor-text-blue": "#0B124F",
+        "mingor-blue-2": "#0B124F",
```
Comment (ezerssss) [inline tailwind.config.js]: mas nindot unta nga 

```
"mingor-blue": {
    100: "#14194A",
    200: "#0B124F"
}
```

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 5 above)
Comment (maxellmilay) [reply tailwind.config.js]: I agree!

File: src/app/page.tsx (hunk 1 above)
Comment (maxellmilay) [reply src/app/page.tsx]: resolved

File: src/components/Footer.tsx (hunk 2 above)
Comment (maxellmilay) [reply src/components/Footer.tsx]: resolved

File: src/components/Footer.tsx (hunk 3 above)
Comment (maxellmilay) [reply src/components/Footer.tsx]: resolved

File: src/components/Footer.tsx (hunk 4 above)
Comment (maxellmilay) [reply src/components/Footer.tsx]: resolved

File: tailwind.config.js (hunk 5 above)
Comment (maxellmilay) [reply tailwind.config.js]: resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**